src/server/index.js

```diff
@@ -33,6 +36,8 @@ export function Server({ games, db, _clientInfo, _roomInfo }) {
     }
   }
 
+  const apiApp = createApiServer({ db, games });
```

Comment:
What do you think about decoupling the lobby API and game servers? That will give people more flexibility to choose to host them independently. The caller can then look something like:

```
const gameServer =  GameServer({ games });
const lobbyServer = LobbyServer({ games });
Run([
  { server: gameServer, port: 8000 },
  { server: lobbyServer, port: 8001 },
])
```

Reply:
It's a little weird to me that the game and lobby server would take separate arguments for their games. I get the flexibility argument, but at the same time I don't know if we want to guard for cases like "game server was started with different set of games from lobby server".

Reply:
But that's actually a feature, isn't it? For example, you could have one lobby server that's the entry point that can handle all games, and the admin creates one game server per game (for better resource management).